app/app/build.gradle

```diff
@@ -37,6 +40,7 @@ android {
 }
 
 dependencies {
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
You should make an order for all your dependencies. or example you should not add libraries above version variables